indra/newview/llappviewer.cpp

```diff
@@ -3552,7 +3552,7 @@ void LLAppViewer::writeSystemInfo()
 	gDebugInfo["ClientInfo"]["MajorVersion"] = LLVersionInfo::instance().getMajor();
 	gDebugInfo["ClientInfo"]["MinorVersion"] = LLVersionInfo::instance().getMinor();
 	gDebugInfo["ClientInfo"]["PatchVersion"] = LLVersionInfo::instance().getPatch();
-	gDebugInfo["ClientInfo"]["BuildVersion"] = LLVersionInfo::instance().getBuild();
+	gDebugInfo["ClientInfo"]["BuildVersion"] = STRINGIZE(LLVersionInfo::instance().getBuild());
```

Comment:
Please use lowercase `stringize()` instead of the `STRINGIZE()` macro for new usage.

`STRINGIZE()` predates (type-safe) support for variadic functions. It accepts a partial `operator<<()` expression, which works but is eyebrow-raising for future maintainers. Besides, it upsets Dr. Stroustrup.

`stringize()` is a proper variadic function.

For a single argument like this, you could alternatively use `std::to_string()`.

Reply:
good point, thanks